### DISCUSSION TITLE
Time Machine

### DISCUSSION THREAD

**Discussion by ElliseY** [jacklandrin/OnlySwitch#62] [Ideas]
Hello Jacklandrin,

Could you please develop a switch on Time Machine?

So it becomes possible to switch on/off the time machine in the menu bar.
<img width="419" alt="2022-12-13 1 30 54" src="https://user-images.githubusercontent.com/106649419/207235257-ebc47c13-d331-4747-84d9-da0dd0bbe9bf.png">

Please help out those who use Time Machine!

Many thanks!!

**Comment by jacklandrin** [comment]
Thank you for your suggestion. I will consider it.
